sickbeard/providers/torrentday.py

```diff
@@ -104,7 +104,13 @@ def search(self, search_strings, age=0, ep_obj=None):  # pylint: disable=too-man
                     self.session.cookies.clear()
                     continue
 
-                torrents = jdata.get('Fs', [dict()])[0].get('Cn', {}).get('torrents', [])
+                try:
+                    torrents = jdata.get('Fs', [dict()])[0].get('Cn', {}).get('torrents', [])
+                except Exception:
```

Comment:
its a temp adding to catch this issue:
https://github.com/pymedusa/SickRage/issues/728